[Bugfix] Fix dead links in Readme (#34)
eclipse-velocitas/vehicle-app-python-template: README.md

This repository provides you with a complete development environment for your own Vehicle App based on the [Software defined vehicle platform](https://sdv.eclipse.org/) including a sample Vehicle App using the [Vehicle App Python SDK](https://github.com/eclipse-velocitas/vehicle-app-python-sdk). The development environment uses the [Development Container](https://code.visualstudio.com/docs/remote/create-dev-container#:~:text=%20Create%20a%20development%20container%20%201%20Path,additional%20software%20in%20your%20dev%20container.%20More%20) feature of Visual Studio Code.

## Documentation

> * [Velocitas Development Model](https://eclipse-velocitas.github.io/velocitas-docs/docs/about/development_model/)

Reviewer: ```suggestion
* [Velocitas Development Model](https://websites.eclipseprojects.io/velocitas/docs/about/development_model/)
```

> * [Vehicle App SDK Overview](https://eclipse-velocitas.github.io/velocitas-docs/docs/about/development_model/vehicle_app_sdk/)

Reviewer: ```suggestion
* [Vehicle App SDK Overview](https://websites.eclipseprojects.io/velocitas/docs/about/development_model/vehicle_app_sdk/)
```

## Quickstart Tutorials
1. [Setup and Explore Development Enviroment](https://eclipse-velocitas.github.io/velocitas-docs/docs/tutorials/quickstart/)

> 1. [Develop your own Vehicle Model](https://eclipse-velocitas.github.io/velocitas-docs/docs/tutorials/tutorial_how_to_create_a_vehicle_model/)

Reviewer: ```suggestion
1. [Develop your own Vehicle Model](https://websites.eclipseprojects.io/velocitas/docs/tutorials/tutorial_how_to_create_a_vehicle_model/)
```

> 1. [Develop your own Vehicle App](https://eclipse-velocitas.github.io/velocitas-docs/docs/tutorials/vehicle-app-development/)

Reviewer: ```suggestion
1. [Develop your own Vehicle App](https://websites.eclipseprojects.io/velocitas/docs/tutorials/vehicle-app-development/)
```

## Contribution
- [GitHub Issues](https://github.com/eclipse-velocitas/vehicle-app-python-template/issues)
- [Mailing List](https://accounts.eclipse.org/mailing-list/velocitas-dev)

> - [Contribution](https://websites.eclipseprojects.io/velocitas/docs/contribution-guidelines)

Reviewer: We should reference here the local CONTRIBUTING.md

> - [Contribution](https://eclipse-velocitas.github.io/velocitas-docs/docs/contribution-guidelines)

Reviewer: ```suggestion
- [Contribution](https://websites.eclipseprojects.io/velocitas/docs/contribution-guidelines)
```
